pkg/scheduler/scheduler_test.go

```diff
@@ -2658,6 +2695,17 @@ func TestSchedule(t *testing.T) {
 					t.Errorf("unexpected events (-want/+got):\n%s", diff)
 				}
 			}
+
+			for cqName, want := range tc.wantSkippedPreemptions {
+				val, err := testutil.GetGaugeMetricValue(metrics.AdmissionCyclePreemptionSkips.WithLabelValues(cqName))
```

Comment:
is there a straightforward way to transform this to a map - then we can do a simple diff (and omit 0 values in the test case definition)?

Reply:
Nothing directly available in `k8s.io/component-base/metrics/testutil`